### DISCUSSION TITLE
Steam support?

### DISCUSSION THREAD

**Discussion by bopthoughts** [sebanc/brunch#1492] [General]
Will brunch enable support for borealis steam?

**Comment by JuliaAgostino** [comment]
> Will brunch enable support for borealis steam?

I hope so! 🤞

**Comment by ycavan** [comment]
Steam works in Brunch.  I installed Brunch on an 11th gen intel laptop, put it developer mode, typed the "insert_coin" command in crosh and installed steam.

**Reply by glblanco** [reply]
did you put the computer on developer mode? or did you change the channel to development?

because the 2nd option is not working for me

**Comment by ycavan** [comment]
I put that laptop into the developer channel under About Chrome OS -> Additional details -> Channel: Change channel.
Remember that you need to make sure that you selected "Allow Updates" when you set up Brunch.
The other thing is that Google specifically mentioned Volteer as the recovery they set this up on, what recovery are you using?
    On Monday, April 4, 2022, 06:18:18 PM CDT, glblanco ***@***.***> wrote:  
 
 


did you put the computer on developer mode? or did you change the channel to development?

because the 2nd option is not working for me

**Reply by glblanco** [reply]
using volteer. will try it tomorrow
